app/static/src/app/components/options/ParameterValues.vue

```diff
@@ -86,6 +93,16 @@ export default defineComponent({
             return (!fitTabIsOpen.value || paramsToVary.value.length) ? "" : userMessages.modelFit.selectParamToVary;
         });
 
+        const odinParameters = computed(() => store.state.model.odinModelResponse.metadata.parameters);
+
+        const reset = () => {
+            const defaultParams: OdinUserType = {};
+            odinParameters.value.forEach((param: OdinParameter) => {
+                defaultParams[param.name] = param.default || 0;
+            });
+            store.commit(`run/${RunMutation.UpdateParameterValues}`, defaultParams);
+        };
```

Comment:
I still think it would be good to only call the update if anything has actually changed so we don't get spurious 'update required' message. Something like this?

```suggestion
        const reset = () => {
            const defaultParams: OdinUserType = {};
            odinParameters.value.forEach((param: OdinParameter) => {
                const defaultValue = param.default || 0;
                if (paramValues.value[param.name] !== defaultValue) {
                    defaultParams[param.name] = defaultValue;
                }
            });
            if (Object.keys(defaultParams).length) {
                store.commit(`run/${RunMutation.UpdateParameterValues}`, defaultParams);
            }
        };
```